sonar-api-model/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
```

```suggestion
    id("io.codemodder.java-library")
    id("io.codemodder.maven-publish")
```

we have convention plugins for these which helps standardize our usage inside this project